dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/OpenAI/Models/Assistant.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Microsoft.Teams.AI.AI.OpenAI.Models
+{
+    internal class Assistant
+    {
+        [JsonPropertyName("id")]
+        public string Id { get; set; } = string.Empty;
+
+        [JsonPropertyName("created_at")]
+        public long CreatedAt { get; set; }
+
+        [JsonPropertyName("description")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```

Comment:
Can pass the condition when serializing/deserializing for all null values?

Reply:
We do have global settings at https://github.com/microsoft/teams-ai/blob/breaking-changes-2/dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/OpenAI/OpenAIClient.cs#L30
Here's another callout in case it's used out of `OpenAIClient`.

BTW, it does not impact the functionality, just make the output JSON prettier.